js/pages/results_framework/components/level_list.js

```diff
@@ -103,6 +107,28 @@ export class LevelListPanel  extends React.Component {
                 </div>;
 
         }
+        
+        let bulkImportBanner = 
+            <div id="bulk-import-banner-alert" className="alert fade show" role="alert">
+                <div className="bulk-alert-message">
+                    <div className="bulk-alert-icon">
+                        <i className="fas fa-bullhorn"></i>
+                    </div>
+                    <div className="bulk-alert-text">
+                        <span>
+                            {
+                                // # Translators: A alert to let users know that instead of entering indicators one at a time, they can use an Excel template to enter multiple indicators at the same time. First step is to build the result framework below, then click the 'Import indicators' button above
+                                gettext('Instead of entering indicators one at a time, use an Excel template to import multiple indicators! First, build your result framework below. Next, click the “Import indicators” button above.')
+                            }
+                        </span>
+                    </div>
+                </div>
+                <button type="button" className="close" data-dismiss="alert" aria-label="Close" onClick={ this.hideBanner }>
+                    <span aria-hidden="true" className="x-modal">&times;</span>
+                </button>
+            </div>
+        
+        let hideBanner = sessionStorage.getItem('hide_bulk_import_alert');
```

Comment:
I don't think they conflict, but you might want to name one of the method above or this variable differently, just for clarity.

Reply:
Updated the click handle method's name.